src/gui/presets.c

```diff
@@ -212,26 +198,28 @@ static void menuitem_delete_preset(GtkMenuItem *menuitem, dt_iop_module_t *modul
   }
   g_free(name);
 }
+static void edit_preset_final_callback(dt_gui_presets_edit_dialog_t *g)
```

Comment:
as we are at it, all static routines should start with underscore. would be nice to fix all instances in this file.